### DISCUSSION TITLE
Bug: Audio choppy after export

### DISCUSSION THREAD

**Discussion by Jboten** [keiwando/toonsquid#309] [Q&A] [answered]
I’m working on a short intro section to an animated music video, and I have a wind noise audio clip that comes in during the second sequence of title cards. The audio plays fine while previewing in the app, but upon export it becomes choppy, as though heard through fan blades. I’ve tried re-exporting, exporting after clearing cache, exporting after iPad reset, export after moving the starting frame of the audio, but none of that seems to solve the issue. It seems only to occur with this particular clip as other sounds seem fine. it was also working in previous exports.

Both the project and export are too large to attach so [here’s](https://www.dropbox.com/s/0n4n78khmwj8hbu/Grave%20dance.mp4?dl=0) a link to the export.  The wind is unexpectedly choppy during the “grave dance” title card. [Here’s](https://www.dropbox.com/s/jxqkcqc9vi570dc/Grave%20dance.tsproj?dl=0) a link to the project.

**Comment by Jboten** [comment]
Update: I had a fade in on the clip which wasn’t wholly necessary, I removed the the fade in and the choppiness went away. So I guess the issue is related to the fade in via key framing the volume up.

**Comment by keiwando** [comment]
Thanks for the report and the insights. I will look into it!

**Comment by keiwando** [comment, marked as answer]
This issue should be fixed now with the 1.9.3 update.

**Reply by Jboten** [reply]
Thanks!! Tested today and it is now resolved. Thanks for such an amazing product!
